compiler/luci-interpreter/src/core/Tensor.cpp

```diff
@@ -59,6 +63,8 @@ void Tensor::resize(const Shape &new_shape)
   _shape = new_shape;
   const size_t element_size = getDataTypeSize(_element_type);
   const int32_t num_elements = _shape.num_elements();
+  if (num_elements <= 0)
+    return;
```

Comment:
Could you please add a comment near the `_data` field that it will be nullptr for empty tensors?